src/main/java/io/tackle/pathfinder/services/AssessmentSvc.java

```diff
@@ -29,17 +39,65 @@
     @Transactional
     public AssessmentHeaderDto createAssessment(@NotNull Long applicationId) {
         long count = Assessment.count("application_id", applicationId);
-        log.info("Assessments for application_id [ " + applicationId + "] : " + count);
+        log.info("Assessments (" + LocalTime.now() + ") for application_id [ " + applicationId + "] : " + count);
```

Comment:
Is it necessary to add `LocalTime.now()`? In fact, I'm seeing that many of the `log` messages have the `INFO` level and it means that the production container will continually print every single `log.info()` to the log which might somehow dirty the log output. What do you think? should we change the level of the log from `INFO` to maybe `DEBUG`? if so, should we do it in this PR or in another PR?

Reply:
You are right, no need for the LocalTime.now as the log already has it. This came from an old System.out.println
I will remove these logs
:+1: